### ISSUE TITLE
Connection to Aleertmanager shows Proxy Error, even though proxy is NOT enabled

### ISSUE THREAD

**Issue by froemra** [HenriWahl/Nagstamon#944]
Migrated from Version 3.10.1 to 3.12.0

The Proxy Handling seems totally garbled.

I have 4 Monitoring Servers configured. All 3 altermanager configurations show "PROXY ERROR", even though there is NO proxy enabled in the settings.
Icinga Monitoring works (proxy disabled as well)...

**Comment by froemra** [comment]
A complete reinstall (Unistall in Windows Apps, manually remove .nagstamon-Directory) and setting up from scratch didn't help either.

**Comment by froemra** [comment]
Finally did a complete uninstall (including .nagstamon-dir) and reinstalling the old Nagstamon-3.10.1-win64_setup.exe -> Everything is working again!

**Comment by HenriWahl** [comment]
Hi @froemra - this sounds really strange. Could you try to run the debug version from ZIP file in parallel - needs no install. Find it at https://github.com/HenriWahl/Nagstamon/releases/download/v3.12.0/Nagstamon-3.12.0-win64_debug.zip

**Comment by HenriWahl** [comment]
@stearz do you have any idea what could cause this?

**Comment by froemra** [comment]
Perhaps Environment-Variables are picked up with the latest Version, no matter what is configured? (HTTPS_PROXY=...)

**Comment by HenriWahl** [comment]
@froemra no, the HTTP_PROXY variable won't be evaluated if no proxy was set. Does it exist at all in your Windows environment?

**Comment by HenriWahl** [comment]
@froemra aside the previous question - is there any network security software active in your network?

**Comment by froemra** [comment]
> @froemra no, the HTTP_PROXY variable won't be evaluated if no proxy was set. Does it exist at all in your Windows environment?

Yes, HTTP_PROXY, HTTPS_PROXY and NO_PROXY are set in System-Env

**Comment by froemra** [comment]
> @froemra aside the previous question - is there any network security software active in your network?

Well sure, there are Firewalls etc., but nothing local on my Laptop that I know of. A college upgraded using the same Configuration without Problems, so there has to be something with my local configuration ...

**Comment by HenriWahl** [comment]
@froemra are the proxy variables set as well on the Windows machines of your colleague?

**Comment by froemra** [comment]
> @froemra are the proxy variables set as well on the Windows machines of your colleague?

I got it running now! I added the Adress of  Altermanager to the NO_PROXY Environment Variable and it is working... But the Usage of Proxy is still **not** enabled in Nagstamon, so it is definately using the PROXY anyway...

**Comment by HenriWahl** [comment]
@froemra this is really strange. I fear the Alertmanager code handles it a little bit differently due to the fact it is not derived from Nagios behaviour.

@stearz do you have any idea?
